prisma/schema.prisma

```diff
@@ -6,3 +6,147 @@ datasource db {
   provider = "postgresql"
   url      = env("DATABASE_URL")
 }
+
+enum USER_TYPES {
+  CUSTOMERS
+  VENDORS
+}
+
+enum RECIEPT_TYPES {
+  INVOICE
+  BILL
+}
+
+enum RECIEPT_STATUS {
+  PENDING
+  DONE
+}
+
+model LoginUsers {
```

Comment:
Get this with [AuthJS](https://authjs.dev/). So this library creates this login user table.

Reply:
@abdulwahidgul24085 AuthJS [does not support ](https://authjs.dev/reference#roadmap) express/nodejs backend at the moment. Please see the screenshot below. We can use [PassportJS](https://www.passportjs.org/) for this.
![image](https://github.com/Techlabs-do/aftermarket-backend-v2/assets/16741599/c1f76a00-0395-454c-8124-5c993cec597b)

Reply:
Thanks @ibadsiddiqui. @shanialy use PassportJS. We need a simple email and password login, and a middleware for authenticated routes: no password reset, or sign-up. We will sign up using the postman with a custom header. So no one else can sign up. For the following routes add a header `secret-key: techlabs-dev-team`.
1. Sign-up
2. password reset

This will give you enough security so anyone with the route can not use it. Since they don't know the custom header check that we have, if you have any questions, drop me a message. 
P.S.
Add the custom header key in the channel and pin it for other people to use.